internal/apis/config/webhook/types.go

```diff
@@ -27,11 +27,11 @@ type WebhookConfiguration struct {
 
 	// securePort is the port number to listen on for secure TLS connections from the kube-apiserver.
 	// Defaults to 6443.
-	SecurePort *int
+	SecurePort int32
 
 	// healthzPort is the port number to listen on (using plaintext HTTP) for healthz connections.
 	// Defaults to 6080.
-	HealthzPort *int
+	HealthzPort int32
```

Comment:
suggestion (non-blocking): I think from the rest of this PR that `0` is a special value meaning "pick a random port" - if so, would it be worth documenting that here and in the v1alpha1 file?

Reply:
I'll add a comment.

Reply:
https://github.com/cert-manager/cert-manager/pull/6275/commits/db1fcdabb17ccd852a312de0ad7255a36ce25e64